tests/acceptance/07_packages/package_module_interpreter.cf

```diff
@@ -20,7 +20,7 @@ body package_module test_module
     query_installed_ifelapsed => "60";
     query_updates_ifelapsed => "14400";
     default_options => { "$(G.testfile)" };
-    interpreter => "/bin/sh";   # NOTE: no hashbang in the test_module script
+    interpreter => "$(G.sh)";   # /bin/sh; NOTE: no hashbang in the test_module script
```

Comment:
Does this fix something for windows then? Interesting. And I guess it would be... powershell or still /bin/sh since your are running in msys?

Reply:
yes, on Linux it's `/bin/sh`, but on msys it's `/usr/bin/sh` :)